engine/Assets/Tests/PlayMode/TestModuleLoader.cs

```diff
@@ -1,121 +1,121 @@
-﻿using System.Collections;
-using NUnit.Framework;
-using UnityEngine;
-using UnityEngine.TestTools;
-using Engine.ModuleLoader;
-using System.IO;
-using System.IO.Compression;
-using System;
-using UnityEngine.SceneManagement;
-using SynthesisAPI.AssetManager;
-
-namespace Tests
-{
-    public class TestModuleLoader : IPrebuildSetup
```

Comment:
Add this test back and make it work.

Reply:
So I can't replicate the axis error. Could it be that your using the wrong module? Because I did get that error when testing and it was caused by an exception being thrown inside the Setup for sample system.

Reply:
Ok just re-added the test and for whatever reason the test finally determined that Api.dll was a thing.

Reply:
Also regarding the Axis error, did you add the Test.glb file to the module?

Reply:
I will take another look 👍